openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -65,12 +66,16 @@ def convert_empty_to_None(value):
         """ Helper method to convert empty string to None (other values pass through). """
         return None if value == "" else value
 
-    def get_profile_image(self, obj):
+    def get_profile_image(self, user_profile):
         """ Returns metadata about a user's profile image. """
-        data = {'has_image': obj.has_profile_image}
+        data = {'has_image': user_profile.has_profile_image}
         data.update({
             '{image_key_prefix}_{size}'.format(image_key_prefix=PROFILE_IMAGE_KEY_PREFIX, size=size_display_name):
-            get_profile_image_url_for_user(obj.user, size_value)
+            get_profile_image_url_for_user(user_profile.user, size_value)
             for size_display_name, size_value in PROFILE_IMAGE_SIZES_MAP.items()
         })
         return data
+
+    def get_requires_parental_consent(self, user_profile):
```

Comment:
It seems that SerializerMethodField expects to be calling a method on the serializer. I tried changing this to declare "requires_parental_consent" as just a field, but that throws an exception that indicates that it is looking for it to be a true database field:

    Non-existant field 'requires_parental_consent' specified in `read_only_fields` on serializer 'AccountLegacyProfileSerializer'.

I'm going to leave it as is unless there's another way that I'm missing.